src/components/autocomplete/js/autocompleteController.js

```diff
@@ -207,6 +207,17 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     elements.$    = getAngularElements(elements);
   }
 
+	/**
+	 * Finds the correct element that will determine the width
+	 * @returns {*}
+	 */
+	function getWrapTarget() {
+    for (var element = $element; element.length; element = element.parent()) {
```

Comment:
`querySelector` regretfully won't allow me to search just the ancestors. This was modeled after the `getSnapTarget ()` in this same directive.